### ISSUE TITLE
reports that some processes continue running even after the tab has been closed

### ISSUE THREAD

**Issue by r0mflip** [microsoft/terminal#2232]
# Environment
```none
Windows build number: 10.0.18362.239
Windows Terminal version: 0.3.2142.0
Any other software?
* Clink v0.4.9 [git:2fd2c2] (setup to autoinject, http://mridgers.github.io/clink)
* clink installed by using scoop
```

# Steps to reproduce
1. Install clink from `cmd`
  ``` bash
  $ clink autorun install
  ```
2. Configure terminal default profile to cmd `"defaultProfile" : "{0caa0dad-35be-5f56-a8ff-afceeeaa6101}"`
3. Open and close Terminal (UI close button, don't use `exit` command)

This problem didn't exist in previous public preview build.

# Expected behavior
Terminal to exit normally with clink setup.

# Actual behavior
Default `cmd` profile in Terminal with [clink](https://mridgers.github.io/clink/) setup to autorun, Terminal fails to exit properly and runs in background even after closing.

* Terminal with clink autorun setup and running normally:
  ![image](https://user-images.githubusercontent.com/37770235/62421558-2fe52300-b6c1-11e9-96ae-acfc08e0f605.png)

* Terminal closed normally (UI close button), still running:
  ![image](https://user-images.githubusercontent.com/37770235/62421542-05936580-b6c1-11e9-9c23-b044c535ff3e.png)

When the command `exit` is used, the terminal exits without the above problem.

**Comment by lksilesian** [comment]
Same behavior here. Also try opening dynamic port forward - it did stuck after closing terminal and started consuming cpu cycles, but i'm unable to reproduce.

Since yesterday after exiting last session i do ps ax to list running consoles and kill the ones lingering. (I think I got ps from GIT for windows).

**Comment by DHowett-MSFT** [comment]
These things should be helped by #2198.

**Comment by r0mflip** [comment]
Looks like this is fixed in tag `v0.4.2382.0` 🎉 by #2198. Closing for now.
